### PR TITLE
Remove 4.5 testing and fix 4.9

### PR DESCRIPTION
**Pull request by kpouget**

### CODE REVIEW

**Comment by kpouget** [comment]
\cc @omertuc

File: ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml [hunk 1]
```diff
@@ -344,6 +344,133 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )4.8-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (omertuc) [inline ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml:348]: optional, always_run

**Comment by omertuc** [comment]
Git rename heuristics are silly sometimes

File: ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml (hunk 1 above)
Comment (kpouget) [reply ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml]: fixed for `always_run false`, what's `optional: true`?

Comment (omertuc) [reply ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml]: It means that even when you run the job manually, and it fails, it still doesn't block a merge

**Comment by kpouget** [comment]
> /hold maybe you want to rehearse

let's wait for 4.9 test to complete, mostly out of curiosity

**Comment by kpouget** [comment]
/hold cancel

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/20681/rehearse-20681-pull-ci-openshift-psap-ci-artifacts-master-4.9-gpu-operator-e2e/1420707470753402880/artifacts/gpu-operator-e2e/presubmit-operatorhub/artifacts/005__nfd_operator__deploy_from_operatorhub/_ansible.log

we'll solve/study that later

**Pull request merged**